README.md

```diff
@@ -39,12 +35,12 @@ When a repository is cloned, it has a default remote called `origin` that points
 
    `git fetch upstream`
 
-   `git merge upstream/master`
+   `git rebase upstream/develop`
 
    This will give you an exact copy of the current remote, make sure you don't have any local changes.
 
 ## Contributing and developing a feature
-1. Make sure you are in the master branch `git checkout master`
+1. Make sure you are in the develop branch `git checkout develop`
 1. Sync your copy `git pull`
```

Comment:
Can you remove the 3rd line which include `git pull`. As it's already mentioned above for fetching & rebasing, it's redundant.